test/benchmark/tests/utils/report.py

```diff
@@ -123,19 +123,21 @@ def add_newline(self):
     def get_document(self):
         return self.markdown_content
 
-def main(s3_bucket_uri):
+def generate_comprehensive_report(s3_bucket_uri):
     """
     Compile a markdown file with different csv files as input
     """
     # Download the s3 files
-    run(f"mkdir -p /tmp/report")
-    run(f"aws s3 cp --recursive {s3_bucket_uri} /tmp/report")
+    # run(f"mkdir -p /tmp/report")
```

Comment:
Do we need these commented lines?

Reply:
We do - currently 'report.py' needs to be run separately from automation. I've added a task here to automate that as well: https://github.com/pytorch/serve/issues/1121

Reply:
Fixed in the current PR now.